alerts.go

```diff
@@ -55,14 +64,38 @@ func (c *Client) FindAlerts() ([]*Alert, error) {
 		return nil, err
 	}
 
-	var data struct {
-		Alerts []*Alert `json:"alerts"`
-	}
-	err = json.NewDecoder(resp.Body).Decode(&data)
+	err = json.NewDecoder(resp.Body).Decode(&d)
 	if err != nil {
 		return nil, err
 	}
-	return data.Alerts, err
+	return &d, err
+}
+
+// FindAlerts find open alerts
+func (c *Client) FindAlerts() (*AlertsResp, error) {
+	return c.findAlertsWithParam(url.Values{})
+}
+
+// FindAlertsByNextID find next open alerts by next id
+func (c *Client) FindAlertsByNextID(nextID string) (*AlertsResp, error) {
+	v := url.Values{}
+	v.Set("nextId", string(nextID))
```

Comment:
Oh, this casting by `string()` is not necessary anymore.

Reply:
It fixed.